x/observer/migrations/v10/migrate_test.go

```diff
@@ -0,0 +1,100 @@
+package v10_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	keepertest "github.com/zeta-chain/node/testutil/keeper"
+	"github.com/zeta-chain/node/testutil/sample"
+	"github.com/zeta-chain/node/x/observer/keeper"
+	v10 "github.com/zeta-chain/node/x/observer/migrations/v10"
+	"github.com/zeta-chain/node/x/observer/types"
+)
+
+var chainParams = types.ChainParamsList{
+	ChainParams: []*types.ChainParams{
+		makeChainParamsEmptyConfirmation(1, 14),
+		makeChainParamsEmptyConfirmation(56, 20),
+		makeChainParamsEmptyConfirmation(8332, 3),
+		makeChainParamsEmptyConfirmation(7000, 0),
+		makeChainParamsEmptyConfirmation(137, 200),
+		makeChainParamsEmptyConfirmation(8453, 90),
+		makeChainParamsEmptyConfirmation(900, 32),
+	},
+}
+
+func TestMigrateStore(t *testing.T) {
+	t.Run("can migrate confirmation count", func(t *testing.T) {
+		k, ctx, _, _ := keepertest.ObserverKeeper(t)
+
+		// set chain params
+		setChainParamsList(ctx, *k, chainParams)
```

Comment:
Why not use the existing function to set the list

Reply:
used existing function in stead https://github.com/zeta-chain/node/pull/3461/commits/ea3f43b2cf2fc46c60459e7487ce0ddc60000194